### DISCUSSION TITLE
Pcb upgrade V2.0

### DISCUSSION THREAD

**Discussion by ericjunkins** [nasa-jpl/open-source-rover#237] [Ideas]
We've had many ideas on a few different locations now, between tapatalk, github issues, and slack discussion about proposed ideas for improvements to our main pcb. I think we should try and collect them here, and decide what we think should go into the design. 

I'll post some ideas here in the future soon, just getting the discussion started

**Comment by apollokit** [comment]
FYI project for this is here: https://github.com/nasa-jpl/open-source-rover/projects/2

Calling it the PCB rev F
